src/services/LoginService.js

```diff
@@ -6,28 +6,26 @@ and store the returned token.
 import axios from 'axios';
 import { setUserSession } from '../utils/Common';
 
-export async function LoginAPI(email, password) {
-  try {
-    const data = await axios.post('http://localhost:1337/api/auth/local', {
-      identifier: email,
-      password,
-    });
+export function LoginAPI(email, password) {
+  return axios.post('http://localhost:1337/api/auth/local', {
```

Comment:
Try to set base url as constant say - http://localhost:1337 in this case and concatenate with others.